packages/shared-ux/card/no_data/impl/src/services.tsx

```diff
@@ -24,10 +24,10 @@ const Context = React.createContext<Services | null>(null);
  * A Context Provider that provides services to the component and its dependencies.
  */
 export const NoDataCardProvider: FC<NoDataCardServices> = ({ children, ...services }) => {
-  const { addBasePath, canAccessFleet } = services;
+  const { addBasePath, canAccessFleet, customBranding } = services;
 
   return (
-    <Context.Provider value={{ addBasePath, canAccessFleet }}>
+    <Context.Provider value={{ addBasePath, canAccessFleet, customBranding }}>
```

Comment:
Why do we need this for card? Card is not showing any logos or anything.

Reply:
Interesting and thank you!  I'll remove it and see if the errors disappear with refactoring. I was hitting something along the lines of ```Property 'customBranding' is missing in type '{ addBasePath: (path: string) => string; canAccessFleet: boolean; }' but required in type 'Services'.ts(2741)
index.d.ts(23, 3): 'customBranding' is declared here.
index.d.ts(331, 9): The expected type comes from property 'value' which is declared here on type 'IntrinsicAttributes & ProviderProps<Services | null>'```